runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -618,4 +628,8 @@ export default class ExcelActionHandler extends StatefulActionHandler {
         }
         throw new Error(ExcelErrorMessage.tableNotFoundIncorrectInput());
     }
+
+    private isPositiveInteger(number: number): boolean {
```

Comment:
In every occurrence of this function you negate the result. So maybe isNegativeInteager function would be better?

Reply:
Ok, changed it. The only thing is that 0 is neither negative or positive number so isNegative now check if the number is below 0, so 0 is acceptable (previously it wasn't), action will be executed, but nothing will happen. Checking if is positive allowed me to exclude 0 as well.